packages/block-library/src/heading/edit.js

```diff
@@ -34,7 +34,7 @@ function HeadingEdit( {
 	const { TextColor, InspectorControlsColorPanel } = __experimentalUseColors(
 		[ { name: 'textColor', property: 'color' } ],
 		{
-			contrastCheckers: { backgroundColor: true, textColor: true },
+			contrastCheckers: [ { backgroundColor: true, textColor: true } ],
```

Comment:
This one doesn't manage a background color. We don't need the array form.

Reply:
Updated 👍

Reply:
cc: @epiqueras